include/constants/pokemon.h

```diff
@@ -213,4 +213,9 @@
 #define UNOWN_FORM_COUNT 28
 #define BOX_NAME_LENGTH 8
 
+#define EVOTRIGGER_LEVEL_UP    0
+#define EVOTRIGGER_TRADE       1
+#define EVOTRIGGER_STONE       2
+#define EVOTRIGGER_STONE_2     3
```

Comment:
Use the constants from pokeemerald
<https://github.com/pret/pokeemerald/blob/master/include/constants/pokemon.h#L341-L344>